src/main/scala/nl.knaw.dans.easy.dd2d/Command.scala

```diff
@@ -57,8 +57,8 @@ object Command extends App with DebugEnhancedLogging {
         logger.info("Service stopped.")
       }
     })
-    app.start()
     logger.info("Service started ...")
+    app.start()
```

Comment:
Why reverse this?

Reply:
Otherwise you get the info text "Service started" after all the logging that has already taken place.  E.g:

```
[INFO] --- exec-maven-plugin:1.6.0:java (default-cli) @ dd-dans-deposit-to-dataverse ---
[2020-11-11 11:14:52,924] INFO  Checking if validator service can be reached
[2020-11-11 11:14:53,093] INFO  OK: validator service is reachable.
[2020-11-11 11:14:53,094] INFO  Checking if root dataverse can be reached...
[2020-11-11 11:14:53,129] TRACE Dataverse [dvId]: root
[2020-11-11 11:14:53,131] TRACE view [()]: ()
[2020-11-11 11:14:53,133] DEBUG x$6: Request URL = https://ddd.dans.knaw.nl/api/v1/dataverses/root
[2020-11-11 11:14:53,411] TRACE handleResponse [expectedStatus]: WrappedArray(200)
[2020-11-11 11:14:53,414] TRACE prettyPrintJson [()]: ()
[2020-11-11 11:14:53,423] INFO  OK: root dataverse is reachable.
[2020-11-11 11:14:53,425] TRACE start [()]: ()
[2020-11-11 11:14:53,425] INFO  Enqueuing deposits found in inbox...
[2020-11-11 11:14:53,426] DEBUG enqueue: Adding /Users/vesaakerman/git/service/dataverse-dans/dd-dans-deposit-to-dataverse/data/inbox/bag3
[2020-11-11 11:14:53,468] TRACE Deposit [dir]: /Users/vesaakerman/git/service/dataverse-dans/dd-dans-deposit-to-dataverse/data/inbox/bag3
[2020-11-11 11:14:53,475] ERROR Not a deposit: /Users/vesaakerman/git/service/dataverse-dans/dd-dans-deposit-to-dataverse/data/inbox/bag3 does not contain a deposit.properties file
nl.knaw.dans.easy.dd2d.package$CorruptDepositException: Not a deposit: /Users/vesaakerman/git/service/dataverse-dans/dd-dans-deposit-to-dataverse/data/inbox/bag3 does not contain a deposit.properties file
	at nl.knaw.dans.easy.dd2d.Deposit.checkCondition(Deposit.scala:75)
	at nl.knaw.dans.easy.dd2d.Deposit.<init>(Deposit.scala:37)
	at nl.knaw.dans.easy.dd2d.Inbox.$anonfun$enqueue$1(Inbox.scala:46)
	at nl.knaw.dans.easy.dd2d.Inbox.$anonfun$enqueue$1$adapted(Inbox.scala:43)
	at scala.collection.immutable.List.foreach(List.scala:392)
	at nl.knaw.dans.easy.dd2d.Inbox.enqueue(Inbox.scala:43)
	at nl.knaw.dans.easy.dd2d.InboxWatcher.start(InboxWatcher.scala:43)
	at nl.knaw.dans.easy.dd2d.DansDeposit2ToDataverseApp.$anonfun$start$1(DansDeposit2ToDataverseApp.scala:51)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.Try$.apply(Try.scala:213)
	at nl.knaw.dans.easy.dd2d.DansDeposit2ToDataverseApp.start(DansDeposit2ToDataverseApp.scala:51)
	at nl.knaw.dans.easy.dd2d.Command$.$anonfun$runAsService$1(Command.scala:60)
	at scala.util.Try$.apply(Try.scala:213)
	at nl.knaw.dans.easy.dd2d.Command$.runAsService(Command.scala:52)
	at nl.knaw.dans.easy.dd2d.Command$.$anonfun$result$1(Command.scala:43)
	at scala.util.Success.flatMap(Try.scala:251)
	at nl.knaw.dans.easy.dd2d.Command$.delayedEndpoint$nl$knaw$dans$easy$dd2d$Command$1(Command.scala:36)
	at nl.knaw.dans.easy.dd2d.Command$delayedInit$body.apply(Command.scala:26)
	at scala.Function0.apply$mcV$sp(Function0.scala:39)
	at scala.Function0.apply$mcV$sp$(Function0.scala:39)
	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)
	at scala.App.$anonfun$main$1$adapted(App.scala:80)
	at scala.collection.immutable.List.foreach(List.scala:392)
	at scala.App.main(App.scala:80)
	at scala.App.main$(App.scala:78)
	at nl.knaw.dans.easy.dd2d.Command$.main(Command.scala:26)
	at nl.knaw.dans.easy.dd2d.Command.main(Command.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:282)
	at java.lang.Thread.run(Thread.java:748)
[2020-11-11 11:14:53,476] DEBUG enqueue: Adding /Users/vesaakerman/git/service/dataverse-dans/dd-dans-deposit-to-dataverse/data/inbox/bag1
[2020-11-11 11:14:53,476] TRACE Deposit [dir]: /Users/vesaakerman/git/service/dataverse-dans/dd-dans-deposit-to-dataverse/data/inbox/bag1
[2020-11-11 11:14:53,479] DEBUG Deposit: bagDir = /Users/vesaakerman/git/service/dataverse-dans/dd-dans-deposit-to-dataverse/data/inbox/bag1/audiences
[2020-11-11 11:14:53,491] TRACE DepositIngestTask [(deposit, dataverse)]: (Deposit at /Users/vesaakerman/git/service/dataverse-dans/dd-dans-deposit-to-dataverse/data/inbox/bag1,nl.knaw.dans.easy.dd2d.dataverse.DataverseInstance@675def5)
[2020-11-11 11:14:53,506] TRACE add [t]: DepositIngestTask(Deposit at /Users/vesaakerman/git/service/dataverse-dans/dd-dans-deposit-to-dataverse/data/inbox/bag1,nl.knaw.dans.easy.dd2d.dansbag.DansBagValidator@49002324,nl.knaw.dans.easy.dd2d.dataverse.DataverseInstance@675def5,true)
[2020-11-11 11:14:53,507] DEBUG add: Task added to queue
[2020-11-11 11:14:53,507] INFO  Start processing deposits...
[2020-11-11 11:14:53,509] INFO  Starting inbox monitor...
[2020-11-11 11:14:53,509] INFO  Processing thread ready for running tasks
[2020-11-11 11:14:53,510] TRACE run [()]: ()
[2020-11-11 11:14:53,510] INFO  Ingesting Deposit at /Users/vesaakerman/git/service/dataverse-dans/dd-dans-deposit-to-dataverse/data/inbox/bag1 into Dataverse
[2020-11-11 11:14:53,510] TRACE validateBag [bagDir]: /Users/vesaakerman/git/service/dataverse-dans/dd-dans-deposit-to-dataverse/data/inbox/bag1/audiences
[2020-11-11 11:14:53,511] DEBUG Calling Dans Bag Validation Service with http://localhost:20180/validate?infoPackageType=SIP&uri=file:///Users/vesaakerman/git/service/dataverse-dans/dd-dans-deposit-to-dataverse/data/inbox/bag1/audiences/
[2020-11-11 11:14:53,519] INFO  Service started ...
```

Reply:
OK, I see your point.